.travis.yml

```diff
@@ -10,7 +10,7 @@ matrix:
       after_success:
           - bash <(curl -s https://codecov.io/bash)
     - os: linux
-      env: TEST_SUITE=testtravis CFLAGS="-fprofile-arcs -ftest-coverage" LDFLAGS="-fprofile-arcs"
+      env: TEST_SUITE=testmanuals CFLAGS="-fprofile-arcs -ftest-coverage" LDFLAGS="-fprofile-arcs"
```

Comment:
Shouldn't this be an additional travis run? I.e. not replace the testtravis run on Linux.